magento/adobe-stock-integration
# Fix Escape doesn't close the Licensing confirmation message. Instead the Adobe Stock window is closed

recieve information





### Description (*)
With default we create prompt with input field, and when the prompt opened, element focused on input field but when we hide the input there is no ability to focus on prompt window, in this case we focus on the popup window

### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1419: Escape doesn't close the Licensing confirmation message. Instead the Adobe Stock window is closed 

### Manual testing scenarios (*)
1. Go to **Content - Media Gallery**, click **Search Adobe Stock**
2. Sign In your account
3. Select a **not licensed** image and **Save Preview -  Confirm**
4. Click **Search Adobe Stock** again
5. The previously saved Image Preview is opened
6. Click **License**
7. _License Adobe Stock Images? ..._ message appears, do not click any button
![license](https://user-images.githubusercontent.com/45624059/83872140-13c76d80-a73a-11ea-96ef-5eca906140ec.png)
8. Press **Escape** on your keyboard 

The _Licensing ..._ confirmation window is closed
